src/locales/fr-FR/admin.ftl

```diff
@@ -58,7 +58,7 @@ login-signIn-passwordLabel = Mot de passe
 login-signIn-passwordTextField =
   .placeholder = Mot de passe
 
-login-signIn-signInWithEmail = Connectez-vous en utilisant votre email
+login-signIn-signInWithEmail = Se connecter
```

Comment:
the `en utilisant votre email` is meaningful here because it contrasts against other login methods some installations use, we can look into making the copy conditional on having multiple login methods enabled but currently we need the distinction.

Reply:
Oh, ok I see. in french the phrase is way too long, the text overflows outside of the container.

Reply:
I think we'll take your translation for now and fix the overflow issue, and change this back in the future.

What should the full translation be? The one that was originally there or is there a better one? @gfra54

Thank you!